bio/fastqc/wrapper.py

```
@@ -15,8 +15,10 @@
 extra = snakemake.params.get("extra", "")
 log = snakemake.log_fmt_shell(stdout=True, stderr=True)
 # Define memory per thread (https://github.com/s-andrews/FastQC/blob/master/fastqc#L201-L222)
-mem_mb = int(get_mem(snakemake, "MiB") / snakemake.threads)
-
+mem_overhead_factor = snakemake.params.get("mem_overhead_factor", 0.1)
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)